lib/mayaUsd/utils/utilSerialization.cpp

```diff
@@ -424,7 +424,9 @@ void ensureUSDFileExtension(std::string& filePath)
     const std::string  defaultExt(UsdMayaTranslatorTokens->UsdFileExtensionDefault.GetText());
     const std::string  usdCrateExt(UsdMayaTranslatorTokens->UsdFileExtensionCrate.GetText());
     const std::string  usdASCIIExt(UsdMayaTranslatorTokens->UsdFileExtensionASCII.GetText());
-    if (extension != defaultExt && extension != usdCrateExt && extension != usdASCIIExt) {
+    const std::string  usdPackageExt(UsdMayaTranslatorTokens->UsdFileExtensionPackage.GetText());
+    if (extension != defaultExt && extension != usdCrateExt && extension != usdASCIIExt
```

Comment:
The filename was ending up being "abcd.usdz.usd" and with this added check ensures usdz is passed without modification.